targets/tizenrt-artik053/apps/jerryscript/jerry_main.c

```diff
@@ -320,6 +320,7 @@ static jerry_log_level_t jerry_log_level = JERRY_LOG_LEVEL_ERROR;
 static int
 jerry_cmd_main (int argc, char *argv[])
 {
+  srand ((unsigned) (jerry_port_get_current_time ()));
```

Comment:
the implementation is dummy for this target, too. calling srand does not help here either